cli/src/main/resources/help/site.adoc

```diff
@@ -67,6 +74,12 @@ Cancels the operation to receive state from `NYC`.
 `site clear-push-state-status --cache=myCache` +
 Clears the status of the push state operation for `mycache`.
 
+`site state-transfer-mode --cache=myCache --site=NYC` +
+Retrieves the state transfer mode for the cache `mycache` and remote backup `NYC`
+
+`site state-transfer-mode --cache=myCache --site=NYC --mode=AUTO` +
+Sets the state transfer mode to `AUTO` for the cache `mycache` and remote backup `NYC`
```

Comment:
It seems strange to configure the value by just setting a `--mode=AUTO` flag. Shouldn't we have dedicated commands for get & set? e.g.

```
site state-transfer-mode get --cache=myCache --site=NYC
site state-transfer-mode set --cache=myCache --site=NYC AUTO
```

Reply:
ok